crates/biome_js_analyze/src/analyzers/nursery/no_useless_lone_block_statements.rs

```diff
@@ -145,6 +145,11 @@ fn statement_has_block_level_declaration(statement: &AnyJsStatement, is_module:
 }
 
 fn in_control_structure(block: &JsBlockStatement) -> bool {
+    let syntax_kind = block.syntax().parent().unwrap().kind();
```

Comment:
Usually we don't use `unwrap` in rule impl. You can use `if let` or:

```suggestion
    let Some(parent) = block.syntax().parent() else { return false; };
```